migration/v0.16.x-v0.17.x/migrate-openid.sh

```diff
@@ -9,6 +9,9 @@ set -euo pipefail
 # Restart the master pods so that they mount the latest version of the secret after apply
 "${here}/../../bin/ck8s" ops kubectl sc delete pod -n elastic-system -l role=master
 
+# Wait 60 seconds before updating the config after master pod restart
+sleep 60
+
```

Comment:
Something like this would be a bit nicer
```suggestion
# Wait for master pod to be ready before updating the config
kubectl wait --for=condition=ready pod  -n elastic-system opendistro-es-master-0

```